msc_pygeoapi/connector/elasticsearch_.py

```diff
@@ -276,9 +276,24 @@ def update_by_query(self, query, name):
         :return: `bool` of index update status
         """
 
-        self.Elasticsearch.update_by_query(body=query, index=name)
+        try:
+            self.Elasticsearch.update_by_query(body=query, index=name)
+        except Exception:
```

Comment:
Let's make the exception more explicit and log it.

Reply:
I have updated it to : 
```
        try:
            self.Elasticsearch.update_by_query(body=query, index=name)
        except  exceptions.NotFoundError as err:
            LOGGER.error('Could not update index: {}'.format(err))
            return False
```